scripts/spark_eval/run_gcs_cluster.sh

```diff
@@ -11,19 +13,18 @@ fi
 gcloud beta dataproc clusters create "$GCS_CLUSTER" \
   --zone us-central1-a \
   --master-machine-type n1-standard-4 \
-  --master-boot-disk-size 500 \
+  --master-boot-disk-size 1000 \
   --num-workers ${NUM_WORKERS:-10} \
   --worker-machine-type n1-standard-16 \
   --worker-boot-disk-size 2000 \
-  --image-version 1.2 \
+  --image-version 1.3 \
   --max-age 3h \
   --project broad-gatk-collab
 
 # Run scripts
 for script in "$@"
 do
-  SCRIPT_NAME="$script"
-  source "$script"
+  eval "$script" || exit $?
```

Comment:
This should loudly inform the user that it has elected to keep their cluster alive. This seems like a dangerous change to make to the script for some developers compute costs. Could you start the cluster with a time to live perhaps?

Reply:
Thanks @jamesemery.  The cluster already has a max age of 3 hours. I've added a message to say the cluster won't be deleted immediately.